map/src/test/java/openlr/map/simplemockdb/SimpleMockedLine.java

```diff
@@ -0,0 +1,231 @@
+package openlr.map.simplemockdb;
+
+import com.vividsolutions.jts.geom.*;
+import openlr.map.Line;
+import openlr.map.Node;
+import openlr.map.GeoCoordinates;
+import openlr.map.FormOfWay;
+import openlr.map.FunctionalRoadClass;
+import openlr.map.GeoCoordinatesImpl;
+import openlr.map.InvalidMapDataException;
+import org.geotools.geometry.jts.JTS;
+import org.geotools.geometry.jts.JTSFactoryFinder;
+import org.geotools.referencing.CRS;
+import org.geotools.referencing.crs.DefaultGeocentricCRS;
+import org.geotools.referencing.crs.DefaultGeographicCRS;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.operation.MathTransform;
+import org.opengis.referencing.operation.TransformException;
+
+
+import java.awt.geom.Point2D;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class SimpleMockedLine implements Line {
+
+    private generated.Line xmlLine;
+    private static GeometryFactory factory = JTSFactoryFinder.getGeometryFactory();
+    private static MathTransform wgs84ToCartesian = null;
+    private static MathTransform cartesianToWgs843d = null;
+    private Node startNode;
+    private Node endNode;
+    private List<LineSegment> lineSegments;
+
+    public final LineString getLineString() {
+        Coordinate[] crds = new Coordinate[getShapeCoordinates().size()];
+        crds = (Coordinate[]) getShapeCoordinates().stream().map(shapePoint -> {
+            return toCartesian(shapePoint.getLongitudeDeg(), shapePoint.getLatitudeDeg());
+        }).collect(Collectors.toList()).toArray(crds);
+        return factory.createLineString(crds);
+    }
+
+
+    public static Coordinate toCartesian(double lon, double lat) {
+        final Coordinate to = new Coordinate();
+        try {
+            if (wgs84ToCartesian == null) {
+                wgs84ToCartesian = CRS.findMathTransform(DefaultGeographicCRS.WGS84, DefaultGeocentricCRS.CARTESIAN);
```

Comment:
Can we not just include this at the beginning of the file like with the factory?

`private static MathTransform wgs84ToCartesian = CRS.findMathTransform(DefaultGeographicCRS.WGS84, DefaultGeocentricCRS.CARTESIAN);`

Reply:
I tried to put it like this but it is throwing an exception

Reply:
d76e45176078308ffb744e673ccdcc8a20573ab8